lib/buildFactory.js

```diff
@@ -44,9 +39,19 @@ function getCommitSha(config) {
 
         return user.unsealToken()
             .then(token => config.scm.getCommitSha({
-                scmUrl: pipeline.scmUrl,
+                scmUri: pipeline.scmUri,
                 token
-            }));
+            })
+            .then((sha) => {
+                config.modelConfig.sha = sha;
```

Comment:
Changed the function to return an object with the decorated commit and the SHA.

The decorated commit does not include a SHA.